frontend_lib/src/component/Timeline/Revision.jsx

```diff
@@ -1,43 +1,67 @@
+import Avatar from '../Avatar/Avatar.jsx'
 import React from 'react'
 import classnames from 'classnames'
 import { translate } from 'react-i18next'
 import Radium from 'radium'
 import color from 'color'
 import { revisionTypeList } from '../../helper.js'
+import i18n from '../../i18n.js'
+import { Trans } from 'react-i18next'
 
 // require('./Revision.styl') // see https://github.com/tracim/tracim/issues/1156
 
+const RevisionLabel = props => {
+  const label = (type, status) => {
+    if (type === 'status-update') {
+      return props.revisionType.label(status.label)
+    }
+    return props.revisionType.label
+  }
+
+  return (
+    <span className='revision__data' data-cy={`revision_data_${props.number}`}>
+      <span className='revision__data__nb'>{props.number}</span>
+      <i className={`fa fa-fw fa-${props.revisionType.faIcon} revision__data__icon`} style={{color: props.customColor}} />
+      <Avatar
+        width={'22px'}
+        publicName={props.authorPublicName}
+        style={{display: 'inline-block', marginRight: '5px', title: props.authorPublicName}}
+      />
+      <span className='revision__data__label'>{label(props.revisionType.id, props.status)}</span>
```

Comment:
a change has been done here https://github.com/tracim/tracim/commit/7718e2dd96a246420d8e59603029ee882911f1b3#diff-c381371e93295faf01f252a877db426cL30 to fix the translation of the label.
Please reproduce this fix in your PR

Reply:
@Skylsmoi i'm not sure I understand. If the fix has been done in develop, then the only thing to do is to merge develop into dev branch, no?

Why do you ask to reproduce the fix?

Reply:
because he refactored the entire component code. So the merge cannot be done automatically and it looks like the fix has been missed during merge conflict

Reply:
ok. thanks for the explanation

Reply:
fixed